proposals/NNNN-test-scripts-common-modules-refactoring.md

```diff
@@ -0,0 +1,96 @@
+# Test Scripts - Refactoring of Common modules
+
+* Proposal: [SDL-NNNN](NNNN-test-scripts-common-modules-refactoring.md)
+* Author: [Dmytro Boltovskyi](https://github.com/dboltovskyi)
+* Status: **Awaiting review**
+* Impacted Platforms: [ATF]
+
+## Introduction
+
+ATF Test Scripts use a lot of commonly used functions. Such as:
```

Comment:
@dboltovskyi ,

suggest to rewrite Test Scripts to test scripts
also, it would be better like
ATF test scripts use a lot of functions like: